packages/components/image/src/image.tsx

```diff
@@ -37,59 +43,67 @@ export type ImageProps = Omit<HTMLUIProps<"img">, keyof UseImageProps> &
  *
  * @see Docs https://yamada-ui.com/components/media-and-icons/image
  */
-export const Image = forwardRef<ImageProps, "img">((props, ref) => {
-  let {
-    fallback,
-    src,
-    srcSet,
-    loading,
-    ignoreFallback,
-    crossOrigin,
-    className,
-    fallbackStrategy = "beforeLoadOrError",
-    onError,
-    onLoad,
-    referrerPolicy,
-    size: boxSize,
-    fit: objectFit,
-    ...rest
-  } = props
+export const Image = forwardRef<ImageProps, "img">(
+  ({ size: boxSize, ...props }, ref) => {
+    const [styles, mergedProps] = useComponentStyle("Image", {
+      size: boxSize,
+      ...props,
+    })
```

Comment:
How about this?
```tsx
export const Image = forwardRef<ImageProps, "img">(
  ({ size: boxSize, fit: objectFit, ...props }, ref) => {
    const [css, mergedProps] = useComponentStyle("Image", {
      boxSize,
      objectFit,
      ...props,
    })
```

Reply:
@108yen 
Is this modification from a readability standpoint?

Reply:
@cidkumagai 
I thought replacing `boxSize` with `size` would change the behavior.
I think `fit` needs to do the same.

Reply:
@108yen 
I see, thank you very much!
I will make the correction that way.

Reply:
@cidkumagai 

Please remove `styles`.

```diff tsx
export const Image = forwardRef<ImageProps, "img">(
  ({ size: boxSize, fit: objectFit, ...props }, ref) => {
-    const [styles, mergedProps] = useComponentStyle("Image", {
+    const [css, mergedProps] = useComponentStyle("Image", {
      size: boxSize,
      fit: objectFit,
      ...props,
    })
    let {
      fallback,
      src,
      srcSet,
      loading,
      ignoreFallback,
      crossOrigin,
      className,
      fallbackStrategy = "beforeLoadOrError",
      onError,
      onLoad,
      referrerPolicy,
      ...rest
    } = omitThemeProps(mergedProps)

    ignoreFallback = loading != null || ignoreFallback || !fallback

    const status = useImage({ ...props, ignoreFallback })

-    const css = useMemo(
-      () => ({ boxSize, objectFit, ...styles }),
-      [boxSize, objectFit, styles],
-    )
```

Reply:
@108yen
I've tried to fix it, please review.